editor/components/post-publish-panel/prepublish.js

```diff
@@ -30,13 +30,13 @@ function PostPublishPanelPrepublish( {
 			{ hasPublishAction && (
 				<Fragment>
 					<PanelBody initialOpen={ false } title={ [
-						__( 'Visibility: ' ),
```

Comment:
I think the whitespace should stay in these translation strings because it'll make screen readers, copy pasters, and semantic nitpickers like me happier that it's there. Technically the space is part of the translation here.

(See: https://github.com/WordPress/gutenberg/issues/7842#issuecomment-403747049)